### PR TITLE
[DE] Translation Units

### PR DESCRIPTION
**Pull request by DerSchlupf**
translation unit-section - more pages more problems :D
image
image

### CODE REVIEW

File: translations/units.tex/de.po [hunk 1]
```diff
@@ -288,6 +392,11 @@ msgid ""
 "}\n"
 "\n"
 msgstr ""
+"\\note{8}{\n"
```
Comment (qwrtln) [inline translations/units.tex/de.po:395]:
```suggestion
"\\note{9}{\n"
```
The text doesn't fit in the box.

**Comment by DerSchlupf** [comment]
I think i have to short the text on page 29 a little more. I need time to think about it, because here are important text-passages. If anybody has suggestions for shorting, please feel free :)

File: translations/units.tex/de.po [hunk 2]
```diff
@@ -259,6 +339,18 @@ msgid ""
 "Walls and Gates may also \\pagelink{Walls}{reduce} the damage from  \\svg{unit_ranged} attacks.\n"
 "\n"
 msgstr ""
+"\\subsection*{\\pagetarget{Unittype}{Einheitentypen}}\n"
+"Es gibt drei Typen von Einheiten:\n"
+"\\begin{itemize}\n"
```
Comment (qwrtln) [inline translations/units.tex/de.po:344]:
```suggestion
"\\begin{itemize}[leftmargin=15pt]\n"
```
I'm afraid without breaking some rules it won't be possible try this.

File: translations/units.tex/de.po [hunk 3]
```diff
@@ -306,6 +415,20 @@ msgid ""
 "Archangels} & \\lightcell[1.2]{Manticores, Black Dragons} & \\lightcell[1.2]"
 "{Dread Knights, Ghost Dragons} \\\\ \\end{tabularx} }"
 msgstr ""
+"\\hommtable{13}{ \\begin{tabularx}{\\linewidth}{XXXXX} & "
```
Comment (qwrtln) [inline translations/units.tex/de.po:418]:
```suggestion
"\\hommtable{12.5}{ \\begin{tabularx}{\\linewidth}{XXXXX} & "
```
And this.

**Comment by qwrtln** [comment]
There are two orphans after the change: "bewegen" (in Fernkampfeinheiten) and "Ablagestapel" (end of Neutrale Einheiten). If you manage to get rid of them, everything will fit in nicely.

**Comment by DerSchlupf** [comment]
po4a commit

**Comment by DerSchlupf** [comment]
po4a commit

**Comment by DerSchlupf** [comment]
Last typos fixed and in my Opinion its ready for merge. :)

**Review by qwrtln — APPROVED** [review]

**Pull request merged**